src/ipa_pc/mod.rs

```diff
@@ -201,14 +223,6 @@ impl<G: AffineCurve, D: Digest> InnerProductArgPC<G, D> {
         let batch_check_time = start_timer!(|| "Multi poly multi point batch check: succinct part");
```

Comment:
I'd add the reference to the Boneh et al paper also at the signature of fn succinct_check_individual_opening_challenges().